.github/workflows/aws-deploy.yaml

```diff
@@ -0,0 +1,76 @@
+name: aws-deploy
+
+on:
+  pull_request:
+    branches: '*'
+  push:
+    branches: [main, develop]
```

Comment:
deploying cloudformation from 2 branches may mess you up because one might clobber the other.

Reply:
I'll address this by conditionally deploying the `dev` and `prod` stack groups if the triggering branch is `develop` and `main`, respectively (using something like `if: github.ref == 'refs/heads/develop'`). I'm just trying to stay [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).